### PR TITLE
fix: update script for Store And Forward test cases

### LINKED ISSUE #705: StoreAndForward001 and StoreAndForward002 fails on arm64-security
# 🐞 Bug Report

### Affected Services [**REQUIRED**]

The issue is located in integration-test.

### Is this a regression?
No

### Description and Minimal Reproduction [**REQUIRED**]
StoreAndForward001 and StoreAndForward002 fails on arm64-security due to retry log not found.

## 🔥 Exception or Error
<pre><code>
1 != 2
</code></pre>
<pre><code>
4 != 3
</code></pre>

## 🌍  Your Environment
**Deployment Environment:**
arm64- ubuntu

**EdgeX Version [**REQUIRED**]:**
main

### PR DESCRIPTION
**Pull request by jumpingliu**
fix #705 

Signed-off-by: melody <melody@iotechsys.com>

**If your build fails**  due to your commit message not passing the build checks, please review the guidelines here: https://github.com/edgexfoundry/edgex-go/blob/main/.github/Contributing.md

## PR Checklist
Please check if your PR fulfills the following requirements:

- [ ] I am not introducing a breaking change (if you are, flag in conventional commit message with `BREAKING CHANGE:` describing the break)
- [ ] I am not introducing a new dependency (add notes below if you are)
- [ ] I have added unit tests for the new feature or bug fix (if not, why?)
- [x] I have fully tested (add details below) this the new feature or bug fix (if not, why?)
- [ ] I have opened a PR for the related docs change (if not, why?)
  <link to docs PR>

### CODE REVIEW

File: TAF/testScenarios/integrationTest/UC_end_to_end/export_store_forward.robot [hunk 1]
```diff
@@ -16,28 +16,30 @@ ${LOG_FILE_PATH}          ${WORK_DIR}/TAF/testArtifacts/logs/export_store_and_fo
 *** Test Cases ***
 # PersistOnError=true and StoreAndForward.Enable=true
 StoreAndForward001 - Stored data is exported after connecting to http server
+    [Tags]  test
```
[the remark this answers is not in the document]
Comment (jumpingliu) [reply TAF/testScenarios/integrationTest/UC_end_to_end/export_store_forward.robot]: updated.

File: TAF/testScenarios/integrationTest/UC_end_to_end/export_store_forward.robot [hunk 2]
```diff
@@ -16,28 +16,30 @@ ${LOG_FILE_PATH}          ${WORK_DIR}/TAF/testArtifacts/logs/export_store_and_fo
...
     Sleep  5s  # wait until retry fails
-    Then Found Retry Log 2 Times In app-http-export Logs From ${timestamp}
+    Then Wait Until Keyword Succeeds   2x  5s  Found Retry Log 2 Times In app-http-export Logs From ${timestamp}
     And Start HTTP Server And Received Exported Data Contains ${PREFIX}_GenerateDeviceValue_UINT8_RW
     [Teardown]  Run Keywords  Delete device by name ${device_name}
                 ...           AND  Delete all events by age
                 ...           AND  Terminate All Processes  kill=True
 
 StoreAndForward002 - Stored data is cleared after the maximum configured retires
+    [Tags]  test
```
[the remark this answers is not in the document]
Comment (jumpingliu) [reply TAF/testScenarios/integrationTest/UC_end_to_end/export_store_forward.robot]: updated.

File: TAF/testScenarios/integrationTest/UC_end_to_end/export_store_forward.robot [hunk 3]
```diff
@@ -16,28 +16,30 @@ ${LOG_FILE_PATH}          ${WORK_DIR}/TAF/testArtifacts/logs/export_store_and_fo
...
     ${configurations}  Create Dictionary  Enabled=true  RetryInterval=3s  MaxRetryCount=3
     ${device_name}  Set Variable  store-device-2
     ${timestamp}  get current epoch time
     Given Set ${configurations} For app-http-export On Consul
     And Create Device For device-virtual With Name ${device_name}
     When Get device data by device ${device_name} and command ${PREFIX}_GenerateDeviceValue_INT16_RW with ds-pushevent=yes
     Sleep  12s  # wait until retry fails
-    Then Found Retry Log 3 Times In app-http-export Logs From ${timestamp}
+    Then Wait Until Keyword Succeeds   2x  5s  Found Retry Log 3 Times In app-http-export Logs From ${timestamp}
```
[the remark this answers is not in the document]
Comment (jumpingliu) [reply TAF/testScenarios/integrationTest/UC_end_to_end/export_store_forward.robot]: updated.

File: TAF/testScenarios/integrationTest/UC_end_to_end/export_store_forward.robot [hunk 4]
```diff
@@ -16,28 +16,30 @@ ${LOG_FILE_PATH}          ${WORK_DIR}/TAF/testArtifacts/logs/export_store_and_fo
...
+    [Tags]  test
     ${configurations}  Create Dictionary  Enabled=true  RetryInterval=3s  MaxRetryCount=3
     ${device_name}  Set Variable  store-device-1
     ${timestamp}  get current epoch time
     Given Set ${configurations} For app-http-export On Consul
     And Create Device For device-virtual With Name ${device_name}
     When Get device data by device ${device_name} and command ${PREFIX}_GenerateDeviceValue_UINT8_RW with ds-pushevent=yes
     Sleep  5s  # wait until retry fails
-    Then Found Retry Log 2 Times In app-http-export Logs From ${timestamp}
+    Then Wait Until Keyword Succeeds   2x  5s  Found Retry Log 2 Times In app-http-export Logs From ${timestamp}
```
[the remark this answers is not in the document]
Comment (jumpingliu) [reply TAF/testScenarios/integrationTest/UC_end_to_end/export_store_forward.robot]: updated.

**Review by cherrycl — APPROVED** [review]
LGTM

**Pull request merged**